rtorrent-flood.json

```diff
@@ -0,0 +1,28 @@
+{
+  "name": "rtorrent-flood",
```

Comment:
Same is true for this json file as well please. We need to indent with 4 spaces, you could perhaps use some tool/language to dump this json indenting with 4 spaces at the end